Enabled prune mode for Bitcoin package

quiquee opened the pull request:
A new variable BTC_PRUNE is defined. It can have values 550 or above. This is the number of megabytes that the bitcoin blockchain will take.
Thought for those of us who want to have a pruned bitcoin
As this is incompatible with the txindex option , logic has been added to set txindex=0 when BTC_PRUNE has been set to any value different than 0

vdo commented:
Thanks @quiquee !

Please also add the variable to the manifest: `dappnode_package.json`

quiquee commented:
Oh yes, I have to add it to two places actually, right? to the dappnode_package and to the docker_compose as well . I have configured it as default with prune, not sure if it makes sense to have this as a separate package ... you tell me

quiquee commented:
I see I have renamed the package and changed the config to store the blockchain under a different directory. This does not make sense to me actually, so please feel free to discard it and let me know if you prefer that I just add the new environment as we spoke.

vdo commented on dappnode_package.json:
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "bitcoin.dnp.dappnode.eth",
+  "name": "bitcoinprune.dnp.dappnode.eth",
```
Please, remove the prune references. This is a general bitcoin package, and pruning should be optional.

vdo commented on dappnode_package.json:
```diff
@@ -1,7 +1,7 @@
+  "name": "bitcoinprune.dnp.dappnode.eth",
   "version": "0.1.0",
-  "description": "The bitcoin daemon. Bitcoind is a program that implements the Bitcoin protocol for remote procedure call (RPC) use.",
+  "description": "The bitcoin daemon configured with prune option. Bitcoind is a program that implements the Bitcoin protocol for remote procedure call (RPC) use.",
```
same here

quiquee replied:
Done

vdo commented on dappnode_package.json:
```diff
@@ -12,18 +12,20 @@
     ],
     "volumes": [
-      "bitcoin_data:/root/.bitcoin"
+      "bitcoin_data:/root/.bitcoinprune"
```
Same here

quiquee replied:
this has been fixed in a later commit

vdo commented on docker-compose.yml:
```diff
@@ -10,7 +10,7 @@ services:
             - "BTC_RPCPASSWORD=dappnode"
-            - "BTC_TXINDEX=1"
-            - "BTC_PRUNE=0"
+            - "BTC_TXINDEX=0"
```
Why are you enabling pruning by default? Should be zero, and optional

quiquee replied:
Fixed, I thought you wanted a separate package actually
